test/Listener/AutoloaderListenerTest.php

```diff
@@ -24,7 +24,7 @@ class AutoloaderListenerTest extends AbstractListenerTestCase
      */
     protected $moduleManager;
 
-    protected function setUp()
+    protected function setup(): void
```

Comment:
why this change? if I remember correctly method in phpunit is called `setUp` not, `setup`, the same below

Reply:
fixed